Permanent/Network/AccountEndpoint.swift

```diff
@@ -97,7 +107,9 @@ extension AccountEndpoint: RequestProtocol {
     
     var progressHandler: ProgressHandler? {
         get { nil }
-        set {}
+        // swiftlint:disable unused_setter_value
```

Comment:
Maybe you can use : // swiftlint:disable:next unused_setter_value ,you don't need to re-enable swiftlint warning after use.